### ISSUE TITLE
[Blazor] Large download size

### ISSUE THREAD

**Issue by daniel-p-tech** [dotnet/aspnetcore#22141]
Hi,

I have published Blazor as an Azure Web App using the VS publish process (I'm on the latest version of both products).

However, when inspecting the network tab in Chrome, the size of transferred files seems to be larger than expected. It also appears that the browser is downloading .gz rather than .br files. I'm attaching a few screenshots illustrating the problem.

Thanks!

![image](https://user-images.githubusercontent.com/29497049/82683765-2c526680-9c17-11ea-8838-339de2f9de91.png)

![image](https://user-images.githubusercontent.com/29497049/82683785-33797480-9c17-11ea-9996-44c707cadec9.png)

![image](https://user-images.githubusercontent.com/29497049/82683804-3a07ec00-9c17-11ea-97c0-aa5289df046f.png)

![image](https://user-images.githubusercontent.com/29497049/82683822-40966380-9c17-11ea-873d-882c26e737a4.png)

**Comment by javiercn** [comment]
@daniel-p-tech thanks for contacting us.

Can you provide more details about how/where you are deploying? (Is it azure web apps on windows?)

I just tried to repro this and it works as expected.
![image](https://user-images.githubusercontent.com/6995051/82818837-31e8c000-9e54-11ea-890e-cf2411e6681e.png)

**Comment by daniel-p-tech** [comment]
Yes, it's Azure Web App on Windows (see below). I simply used VS publish tool (as document by screenshots in my original post).

![image](https://user-images.githubusercontent.com/29497049/82849894-a7d64100-9ebf-11ea-8106-380939285ee9.png)

**Comment by javiercn** [comment]
@daniel-p-tech are you deploying a standalone application?

**Comment by daniel-p-tech** [comment]
I have just tried to publish a different project using VS Coummunity 16.6.0 (link to a GitHub repo is below) with the same problem.

https://github.com/daniel-p-tech/BlazorTabs

I have published the project using imported publish file that I downloaded from Azure portal and also using the VS Publish Azure Web App wizard.

![image](https://user-images.githubusercontent.com/29497049/83315680-33651000-a1e7-11ea-85f7-ad5f4fcbca75.png)

Thanks.
